Bottom: f099685e186a34b1834cf692b3610fb05149e3e9
Top:    044a5e01a4cac05bca3c1d0eefaf3d5cc4bfbe54
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:08 +0000

DEBUG: Polygon parentage on


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8041c05..efece45 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2964,13 +2964,14 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
 
   *res = NULL;
 
-#if 0
+#if 1
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
-#endif
+#else
   a_copy = NULL;
   b_copy = NULL;
+#endif
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
